apps/user-office-backend/src/datasources/postgres/records.ts

```diff
@@ -712,15 +708,13 @@ export const createProposalObject = (proposal: ProposalRecord) => {
   );
 };
 
-export const createReviewObject = (review: ReviewRecord) => {
-  return new Review(
-    review.review_id,
-    review.proposal_pk,
-    review.user_id,
-    review.comment,
-    review.grade,
-    review.status,
-    review.sep_id
+export const createInstrumentObject = (instrument: InstrumentRecord) => {
```

Comment:
Maybe I am wrong but is it the `createReviewObject` changed to `createInstrumentObject`?